frontend/src/components/quantum/quantumContent.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import QuantumToolbar from "./quantumToolbar";
+import QuantumCircuit from "./quantumCircuit";
+import {Box} from "@material-ui/core";
+
+export default class QuantumContent extends React.Component {
+
+    render() {
+        return (
+            <Box id="content" style={this.props.innerStyle}>
+                <QuantumToolbar/>
+                <QuantumCircuit/>
+                <Box id="progress">
+                    <Box></Box>
```

Comment:
What's the use of nesting another box here?

Reply:
This is actually imported from qcsimulator, which again I don't really want to touch too much. I haven't looked into what it's actually used for.

Reply:
The box component is being imported from material UI tho.

```javascript
import {Box} from "@material-ui/core";
```

I was just specifically wondering why box component is nested in another box with the inner most box not doing anything

``` javascript
<Box id="progress">
     <Box></Box>
</Box>
```

Reply:
I have no idea why qcsimulator did that. They were actually divs instead of Box, I just converted them for consistency reasons, since Box is basically a div but has more functionality from material-ui

Reply:
Good to know, didn't realise it was QC simulator code